support-policy.md

```diff
@@ -0,0 +1,34 @@
+Support and Stability policy
+============================
+
+With regards to current and future releases the OpenSSL project has adopted the
+following policy:
+
+* Version 3.0 will be supported until 2023-09-07.
+* Version 1.1.1 will be supported until 2023-09-11 (LTS).
+* Version 1.0.2 is no longer supported. Extended support for 1.0.2 to gain
+access to security fixes for that version is
+[available](https://www.openssl.org/support/contracts.html).
+* Versions 1.1.0, 1.0.1, 1.0.0 and 0.9.8 are no longer supported.
+
+We may designate a release as a Long Term Support (LTS) release. LTS releases
+will be supported for at least five years and we will specify one at least every
+four years. Non-LTS releases will be supported for at least two years.
+
+During the final year of support, we do not commit to anything other than
+security fixes. Before that, bug and security fixes will be applied as
+appropriate.
+
+Stability
+---------
+
+No API or ABI breaking changes are allowed in a minor or patch release.
```

Comment:
I'm wondering if we need to specify more about this. Do we need to make rules about this, or do we just rely that things will just work out?

For instance, do we want to say that we expect the OTC to work out the details about what an API change is? You can argue that any change in behavior is an API change, but that's clearly not what is meant here.

What is the procedure that needs to be followed if we want to make an exception to this. If the OTC feels that this might be an API change, does the OTC need to seek approval from the OMC? Or is the OMC responsible for monitoring the PRs and put a hold on it?